Index: marble/data/CMakeLists.txt
===================================================================
--- marble.orig/data/CMakeLists.txt	2011-11-17 20:51:36.000000000 +0100
+++ marble/data/CMakeLists.txt	2011-11-17 20:54:16.000000000 +0100
@@ -266,6 +266,7 @@
     install (FILES icons/hi32-app-marble.png DESTINATION ${ICON_INSTALL_DIR}/hicolor/32x32/apps/ RENAME marble.png)
     install (FILES icons/hi22-app-marble.png DESTINATION ${ICON_INSTALL_DIR}/hicolor/22x22/apps/ RENAME marble.png)
     install (FILES icons/hi16-app-marble.png DESTINATION ${ICON_INSTALL_DIR}/hicolor/16x16/apps/ RENAME marble.png)
+    install (FILES svg/marble-harmattan-launcher.svg DESTINATION ${CMAKE_INSTALL_PREFIX}/share/themes/base/meegotouch/icons/ RENAME marble.svg)
 endif(NOT APPLE AND NOT WIN32 )
 
 FILE (GLOB AUDIO_FILES audio/*.ogg)
Index: marble/src/marble.desktop
===================================================================
--- marble.orig/src/marble.desktop	2011-11-17 20:51:48.000000000 +0100
+++ marble/src/marble.desktop	2011-11-17 20:55:29.000000000 +0100
@@ -1,7 +1,6 @@
 [Desktop Entry]
 Type=Application
-TryExec=marble
-Exec=marble
+Exec=/usr/bin/marble-touch
 Name=Marble
 Name[ast]=Marble
 Name[bg]=Marble
@@ -109,7 +108,5 @@
 GenericName[x-test]=xxDesktop Globexx
 GenericName[zh_CN]=桌面地球仪
 GenericName[zh_TW]=桌面星象儀
-Icon=marble
+Icon=/usr/share/themes/base/meegotouch/icons/marble.svg
 Terminal=false
-Categories=Qt;KDE;Education;Geography;
-X-DocPath=marble/index.html
