Index: src-git/data/CMakeLists.txt
===================================================================
--- src-git.orig/data/CMakeLists.txt	2012-01-29 11:54:45.000000000 +0100
+++ src-git/data/CMakeLists.txt	2012-01-29 12:17:35.000000000 +0100
@@ -258,7 +258,7 @@
 
 if(NOT APPLE AND NOT WIN32)
   if(QTONLY)
-    set(ICON_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/share/icons)
+    set(ICON_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/../../usr/share/icons)
   endif(QTONLY)
     install (FILES icons/hi128-app-marble.png DESTINATION ${ICON_INSTALL_DIR}/hicolor/128x128/apps/ RENAME marble.png)
     install (FILES icons/hi64-app-marble.png DESTINATION ${ICON_INSTALL_DIR}/hicolor/64x64/apps/ RENAME marble.png)
@@ -266,7 +266,7 @@
     install (FILES icons/hi32-app-marble.png DESTINATION ${ICON_INSTALL_DIR}/hicolor/32x32/apps/ RENAME marble.png)
     install (FILES icons/hi22-app-marble.png DESTINATION ${ICON_INSTALL_DIR}/hicolor/22x22/apps/ RENAME marble.png)
     install (FILES icons/hi16-app-marble.png DESTINATION ${ICON_INSTALL_DIR}/hicolor/16x16/apps/ RENAME marble.png)
-    install (FILES svg/marble-harmattan-launcher.svg DESTINATION ${CMAKE_INSTALL_PREFIX}/share/themes/base/meegotouch/icons/ RENAME marble.svg)
+    install (FILES svg/marble-harmattan-launcher.svg DESTINATION /usr/share/themes/base/meegotouch/icons/ RENAME marble.svg)
 endif(NOT APPLE AND NOT WIN32 )
 
 FILE (GLOB AUDIO_FILES audio/*.ogg)
Index: src-git/src/CMakeLists.txt
===================================================================
--- src-git.orig/src/CMakeLists.txt	2012-01-29 12:15:29.000000000 +0100
+++ src-git/src/CMakeLists.txt	2012-01-29 12:16:09.000000000 +0100
@@ -221,7 +221,7 @@
         install(FILES marble.kcfg DESTINATION ${KCFG_INSTALL_DIR})
       else( NOT QTONLY)
         install (TARGETS marble RUNTIME DESTINATION bin )
-        install( FILES marble.desktop  DESTINATION share/applications)
+        install( FILES marble.desktop  DESTINATION /usr/share/applications)
       endif(NOT QTONLY)
       #install (TARGETS marble ${INSTALL_TARGETS_DEFAULT_ARGS} )
     endif (APPLE AND QTONLY)
Index: src-git/src/marble.desktop
===================================================================
--- src-git.orig/src/marble.desktop	2012-01-29 12:20:45.000000000 +0100
+++ src-git/src/marble.desktop	2012-01-29 12:57:23.000000000 +0100
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-Exec=/usr/bin/single-instance /usr/bin/marble-touch
+Exec=/usr/bin/env QML_IMPORT_PATH=/opt/marble/lib/marble/plugins LD_LIBRARY_PATH=/opt/marble/lib /usr/bin/single-instance /opt/marble/bin/marble-touch
 Name=Marble
 Name[ast]=Marble
 Name[bg]=Marble
